app/controllers/api/projects.php

```diff
@@ -882,6 +882,14 @@
             throw new Exception(Exception::PROJECT_NOT_FOUND);
         }
 
+        $uniqueNumbers = [];
+        foreach ($numbers as $number) {
+            if (isset($uniqueNumbers[$number['phone']])) {
+                throw new Exception(Exception::GENERAL_BAD_REQUEST, 'Duplicate phone numbers are not allowed.');
```

Comment:
I think it would be better to have exception defined for this. By doing that we give it custom exception type to which developers can then react properly.

Reply:
This is actually something the validator should have caught. But our validator system doesn't support this. 
And if an error is thrown in the validator we usually throw a general 400 Bad Request error.